preconditions/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.palantir.revapi'
 dependencies {
     api project(':safe-logging')
     compileOnly 'com.google.code.findbugs:jsr305'
```

Comment:
should we consider migrating from `jsr305` `javax.annotation.Nullable` to `org.jetbrains.annotations.Nullable` if we're going to effectively use `org.jetbrains:annotations` everywhere or `org.checkerframework.checker.nullness.qual.Nullable` (though there's the current JDK 11/15/17 checker quandry)?

Reply:
I’d hoped a standard would emerge, but it seems everyone is using their own nullable annotation at this point. Aligning with an ide may make sense as they do a great deal of static analysis and benefit from library adoption, my initial instinct was that the jetbrains annotations may be more feature-rich than we’d like for broad use, but I don’t have a terribly strong opinion.

Reply:
I believe https://jspecify.dev / https://github.com/jspecify/jspecify is meant to be the one annotation specification/implementation to rule them all.